account-decoder-client-types/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "solana-account-decoder-client-types"
+description = "Core RPC client types for solana-account-decoder"
+documentation = "https://docs.rs/solana-account-decoder-client-types"
+version = { workspace = true }
+authors = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+
+[dependencies]
+base64 = { workspace = true }
+bs58 = { workspace = true }
+serde = { workspace = true }
+serde_derive = { workspace = true }
+serde_json = { workspace = true }
+solana-account = { workspace = true }
+solana-pubkey = { workspace = true }
+zstd = { workspace = true, optional = true }
+
+[features]
+zstd = ["dep:zstd"]
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+all-features = true
+rustdoc-args = ["--cfg=docsrs"]
```

Comment:
Are these extra keys necessary here? I don't see them used anywhere else in the monorepo.

Reply:
they were added to the monorepo the other day: https://github.com/anza-xyz/agave/pull/3121

Reply:
I just checked and it doesn't make a difference here. But may as well leave it in imo in case someone comes along later and does something that would benefit from it